models/activities/statistic.go

```diff
@@ -28,7 +28,7 @@ type Statistic struct {
 		Mirror, Release, AuthSource, Webhook,
 		Milestone, Label, HookTask,
 		Team, UpdateTask, Project,
-		ProjectBoard, Attachment int64
+		ProjectColumn, Attachment int64
```

Comment:
Shouldn't it be named BoardColumn?

Reply:
I was just renaming board to column for now as that is already a quite large change. Renaming project to board had a still ongoing discussion with mixed opinions, so I skipped that for now.

Reply:
Oh ok